catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -212,6 +212,17 @@
     String RESOURCE_CHECKSUM = "resource-checksum";
 
     /**
+     * {@link Attribute} that provides URIs for derived formats of the {@literal Metacard.RESOURCE_URI}
+     */
+    String DERIVED_RESOURCE_URI = "resource.dervied-resource-uri";
```

Comment:
Looks like derived is misspelled.  Also, `resource.derived-uri` would be less redundant.